lib/rubocop/cop/style/access_modifier_declarations.rb

```diff
@@ -92,10 +112,17 @@ class AccessModifierDeclarations < Base
           (send nil? {:private :protected :public :module_function} (sym _))
         PATTERN
 
+        # @!method access_modifier_with_attr?(node)
+        def_node_matcher :access_modifier_with_attr?, <<~PATTERN
+          (send nil? {:private :protected :public :module_function}
+            (send nil? {:attr_reader :attr_writer :attr_accessor} _))
```

Comment:
Also `:attr` (https://ruby-doc.org/3.2.2/Module.html#method-i-attr)

```suggestion
            (send nil? {:attr :attr_reader :attr_writer :attr_accessor} _))
```

Reply:
Good catch! I forgot it.